src/Steps/Release/CreateChangelog.php

```diff
@@ -320,4 +347,20 @@ public function commitChanges(OutputInterface $output, Library $library, Version
             $repo->run("commit", array("-m", "Added {$versionName} changelog"));
         }
     }
+
+    /**
+     * @return bool
+     */
+    private function getIncludeUpgradeOnly(): bool
```

Comment:
Sounds a bit counter intuitive to have the getter/setter be private and the actual property be protected.

If `includeUpgradeOnly` is meant to be internal to this class, then maybe just ditch the getter/setter and use the property directly.